man/org.freedesktop.systemd1.xml

```diff
@@ -110,6 +110,8 @@ node /org/freedesktop/systemd1 {
                in  i signal);
       CleanUnit(in  s name,
                 in  as mask);
+      FreezeUnit(in  s name);
+      ThawUnit(in  s name);
```

Comment:
@msekletar promised to look into this ;) That's why I omitted the addition of the freezer stuff until now. But yeah, I think we can merge this.